scripts/pre_push_hook.py

```diff
@@ -396,6 +398,16 @@ def main(args=None):
                     python_utils.PRINT(
                         'Push failed, please correct the linting issues above.')
                     sys.exit(1)
+
+            typescript_check_status = 0
+            if does_diff_include_js_or_ts_files(files_to_lint):
```

Comment:
I see this check twice (line 413), why don't we move this code in the same check? (You can use a newline to present visual separation of two checks!)

Reply:
not needed